doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -314,6 +343,36 @@ messages and additional metadata helpful for debugging.
+<!-- | `prepared_statement_id` | [`uuid`]                     | The ID of the prepared statement being executed. Corresponds to [`mz_prepared_statement_history.id`](#mz_prepared_statement_history).                                                                                          | -->
+<!-- | `sample_rate`           | [`double precision`]         | The sampling rate at the time the execution began.                                                                                                                                                                             | -->
+<!-- | `params`                | [`text list`]                | The values of the prepared statement's parameters.                                                                                                                                                                             | -->
+<!-- | `began_at`              | [`timestamp with time zone`] | The time at which execution began.                                                                                                                                                                                             | -->
+<!-- | `finished_at`           | [`timestamp with time zone`] | The time at which execution ended.                                                                                                                                                                                             | -->
+<!-- | `was_successful`        | [`bool`]                     | Whether the statement executed successfully. Mutually exclusive with `was_canceled` and `was_aborted`.                                                                                                                         | -->
+<!-- | `was_canceled`          | [`bool`]                     | Whether the statement execution was canceled. Mutually exclusive with `was_successful` and `was_aborted`.                                                                                                                      | -->
+<!-- p| `was_aborted`           | [`bool`]                     | Whether the statement execution was aborted. "Aborted" means that the execution had not finished when Materialize restarted (for maintenance or due to an error). Mutually exclusive with `was_successful` and `was_canceled`. | -->
+<!-- | `error_message`         | [`test`]                     | The error returned when executing the statement, or `NULL` if it was successful, canceled or aborted.                                                                                                                          | -->
+<!-- | `rows_returned`         | [`uint8`]                    | The number of rows returned by the statement, if it finished successfully and was of a kind of statement that can return rows, or `NULL` otherwise.                                                                            | -->
```

Comment:
Users (including us) don't have a reliable way to get uint8s out of our system (there's no binary encoding that's well understood by drivers). Users could cast it to a postgres `int8`, but it's a bit silly to force that anytime this is queried. Recommend making this an `int8`.